cli/cli/main.go

```diff
@@ -40,8 +46,51 @@ func main() {
 	})
 
 	if err := commands.RootCmd.Execute(); err != nil {
-		// We don't actually need to print the error because Cobra will do it for us
+		maybeCleanedError := getErrorMessageToBeDisplayedOnCli(err)
+		// cobra uses this method underneath as well to print errors
+		// so just used it directly.
+		commands.RootCmd.PrintErrln("Error: ", maybeCleanedError.Error())
```

Comment:
it seems that we miss this sentence `Run '___go_build_github_com_kurtosis_tech_kurtosis_cli_cli --help' for usage.` when we handle the error here.

If I execute `kurtosis wrong-command` in main branch I get this error message:
```
Error: unknown command "wrong-command" for "___go_build_github_com_kurtosis_tech_kurtosis_cli_cli"
Run '___go_build_github_com_kurtosis_tech_kurtosis_cli_cli --help' for usage.

```

but if I run this command on this branch, it miss the second line:
```
Error: unknown command "wrong-command" for "___go_build_github_com_kurtosis_tech_kurtosis_cli_cli"
```

Reply:
oh good catch; i can add that too here. @leoporoli